subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -127,6 +134,7 @@ struct ticker_user_op {
 		struct ticker_user_op_start start;
 		struct ticker_user_op_update update;
 		struct ticker_user_op_slot_get slot_get;
+		s8_t priority;	   /* Node priority. Defaults to 0 */
```

Comment:
This is part of a union, which seems unclear - until one realizes it belongs to TICKER_USER_OP_TYPE_PRIORITY_SET for which we lack a user op struct. 
Would be nice to add that struct for consistency.

Reply:
Will create a struct for it.